test/Swashbuckle.AspNetCore.SwaggerGen.Test/Fixtures/XmlAnnotatedType.cs

```diff
@@ -57,6 +57,12 @@ public class XmlAnnotatedType
         /// <example>d3966535-2637-48fa-b911-e3c27405ee09</example>
         public Guid GuidProperty { get; set; }
 
+        /// <summary>
+        /// Summary for Nullable StringProperty
+        /// </summary>
+        /// <example>null</example>
+        public string NullableStringProperty { get; set; }
```

Comment:
A little nitpick (sorry) but the current name would make me think it's a nullable string in the context of nullable reference types. I think it would make more sense to name it `StringPropertyWithNullExample".



IMO it would make more sense to name this `StringPropertyWithNullExample`.

Reply:
I struggled with how to name this, your suggestion makes a lot more sense. Just updated the PR with this change.